rules/60-hidraw_id.rules

```diff
@@ -0,0 +1,7 @@
+# do not edit this file, it will be overwritten on update
+
+ACTION=="remove", GOTO="hidraw_id_end"
+
+SUBSYSTEM=="hidraw", IMPORT{program}="hidraw_id --udev"
```

Comment:
hmm, isn't there supposed to be a rules addition that adds uaccess to this, too?

(btw, are we sure that giving unpriv clients hidraw access is generally considered safe? is the kernel device API for this designed to be usable by unpriv client?)

Reply:
@amluto I understand you wrote the original code this PR is based on, maybe you can comment on this: is the hidraw kernel subsystem designed to be safe for unpriv userspace to access it? i.e. if we now open access to such devices by default, is that a good and safe idea?

Reply:
> hmm, isn't there supposed to be a rules addition that adds uaccess to this, too?

That rule is already part of systemd, `uaccess` is added for `ID_SECURITY_TOKEN` by `src/login/70-uaccess.rules.m4`. The `ID_U2F_TOKEN` is there solely for full compatibility with the existing `u2f-hidraw-policy` package, which is part of e.g. Fedora.

> (btw, are we sure that giving unpriv clients hidraw access is generally considered safe? is the kernel device API for this designed to be usable by unpriv client?)

I am not knowledgeable enough about its internals to declare it safe, but I can say that (1) having raw access to (correctly designed) security tokens is safe since every meaningful action tests for user presence first and (2) unprivileged clients do already have access to whitelisted security tokens. I would also be interested in @amluto's opinion.

Reply:
> @amluto I understand you wrote the original code this PR is based on, maybe you can comment on this: is the hidraw kernel subsystem designed to be safe for unpriv userspace to access it? i.e. if we now open access to such devices by default, is that a good and safe idea?

As far as I know, this is the intent.  It's been used for exactly this purpose with U2F devices for years, so, if there's a problem, it's not systemd's fault :)